iris_grib/_grib_cf_map.py

```diff
@@ -34,6 +34,7 @@
 
```

Comment:
We have recently dropped Python 2 support and so have been removing any use of six/__future__

Would you be able to also remove lines 30 and 31 as they are no longer needed?